apps/spruce/src/pages/image/GeneralTable/index.tsx

```diff
@@ -0,0 +1,102 @@
+import React, { useMemo, useRef } from "react";
+import { useQuery } from "@apollo/client";
+import { useLeafyGreenTable, LGColumnDef } from "@leafygreen-ui/table";
+import { StyledRouterLink, WordBreak } from "components/styles";
+import { BaseTable } from "components/Table/BaseTable";
+import { getTaskRoute } from "constants/routes";
+import { useToastContext } from "context/toast";
+import {
+  ImageGeneralQuery,
+  ImageGeneralQueryVariables,
+} from "gql/generated/types";
+import { IMAGE_GENERAL } from "gql/queries";
+import { useDateFormat } from "hooks";
+
+type PropertyValue = {
+  property: string;
+  value: React.ReactNode;
+};
+
+type GeneralTableProps = {
+  imageId: string;
+};
+
+export const GeneralTable: React.FC<GeneralTableProps> = ({ imageId }) => {
+  const dispatchToast = useToastContext();
+  const { data: imageData, loading } = useQuery<
+    ImageGeneralQuery,
+    ImageGeneralQueryVariables
+  >(IMAGE_GENERAL, {
+    variables: { imageId },
+    onError(err) {
+      dispatchToast.error(
+        `There was an error loading the image general table: ${err.message}`,
+      );
+    },
+  });
+
+  const getDateCopy = useDateFormat();
+  const image = imageData?.image;
+
+  const tableData = useMemo(() => {
+    if (!image) return [];
+
+    return [
+      {
+        property: "Last deployed",
+        value: image.lastDeployed ? getDateCopy(image.lastDeployed) : "N/A",
+      },
+      {
+        property: "Amazon Machine Image (AMI)",
+        value: image.ami ?? "N/A",
+      },
+      {
+        property: "Latest task",
+        value: image.latestTask?.id ? (
+          <StyledRouterLink to={getTaskRoute(image.latestTask?.id)}>
+            <WordBreak>{image.latestTask?.id}</WordBreak>
+          </StyledRouterLink>
```

Comment:
I don't seem to be able to click this link (this would be a good thing to add to the Cypress tests)

Reply:
I can click the link, but I don't think it's in the database, so it doesn't render the task page.